src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
I just pushed a new version which basically replaces the system call with popen & pclose. The printed output I try to print with comf_printf but that doesn't seem to work. Shouldn't printf_ be used instead? That's what the rest of the code uses as well.

Furthermore I understand you'd rather use strings over the popen pipe to communicate the result of the detected DOS flavor instead of return codes? Do you have any suggestions on what works well for the script to output and the c code to interpret?